src/internet_identity/src/main.rs

```diff
@@ -346,6 +346,7 @@ fn config() -> InternetIdentityInit {
         canister_creation_cycles_cost: Some(persistent_state.canister_creation_cycles_cost),
         register_rate_limit: Some(persistent_state.registration_rate_limit.clone()),
         captcha_config: Some(persistent_state.captcha_config.clone()),
+        related_origins: persistent_state.related_origins.clone(),
```

Comment:
Any reason why all the above are wrapped with `Some` and this one isn't? Apparently the ones above are also optional similar to this one.

Reply:
Edit: nevermind, the persistent state and init aren't equal types.

Reply:
Yes, the rest of the configs are not optional in the state. There is a default captcha for example. I could use a default empty array instead as well, but I thought it was meaningful to be an option of list of origins instead.